templates/TwigBundle/Exception/error404.html.twig

```diff
@@ -2,31 +2,16 @@
 
 {% block banner '' %}
 {% block page_title 'Page introuvable' %}
-{% block stylesheets %}
-    <style type="text/css">
-        header,
-        footer {
-            display: none !important;
-        }
-    </style>
-{% endblock %}
 
 {% block content %}
     <main class="error">
-        <div class="header l__wrapper">
-            <h1>
-                <a href="{{ path('homepage') }}">
-                    En Marche !
-                </a>
-            </h1>
-        </div>
         <section class="l__wrapper--slim">
             <div class="error__box error__box--404"></div>
             <div>
                 <h1>
                     Cette page n'existe pas !
                 </h1>
-                <p class="text--body text--gray">
+                <p class="text--body text--black">
```

Comment:
You don't need `text--black` here since the text is by default black .

Reply:
https://app.clubhouse.io/enmarche/story/557/etq-utilisateur-je-ne-veux-que-la-page-404-ne-soit-pas-un-cul-de-sac-je-veux-pouvoir-revenir-facilement-a-en-marche-et-savoir

Ok for the text--black